persist/fs/persist_manager.go

```diff
@@ -350,3 +566,21 @@ func (pm *persistManager) SetRuntimeOptions(value runtime.Options) {
 	pm.currRateLimitOpts = value.PersistRateLimitOptions()
 	pm.Unlock()
 }
+
```

Comment:
nit: Do you mind deleting the `ensure...` methods here, as discussed these are probably fine since the glob path will succeed when the dir isn't there but just return empty results (which its totally fine).

Reply:
sure thing

Reply:
done